Epoch 0:
	train loss:3574111.6034  val ap:0.952748  val auc:0.949150
	total time:136.08s sample time:0.62s prep time:85.66s
Epoch 1:
	train loss:3078570.0623  val ap:0.958502  val auc:0.955061
	total time:123.39s sample time:0.60s prep time:75.25s
Loading model at epoch 0...
Traceback (most recent call last):
  File "/home/guorui/workspace/dgnn/a-tgl/train.py", line 246, in <module>
    model.load_state_dict(torch.load(path_saver))
  File "/home/guorui/anaconda3/envs/sgnn/lib/python3.8/site-packages/torch/serialization.py", line 699, in load
    with _open_file_like(f, 'rb') as opened_file:
  File "/home/guorui/anaconda3/envs/sgnn/lib/python3.8/site-packages/torch/serialization.py", line 230, in _open_file_like
    return _open_file(name_or_buffer, mode)
  File "/home/guorui/anaconda3/envs/sgnn/lib/python3.8/site-packages/torch/serialization.py", line 211, in __init__
    super(_open_file, self).__init__(open(name, mode))
FileNotFoundError: [Errno 2] No such file or directory: 'models/TALK_1721918417.9247391.pkl'
